config/locales/fr.yml

```diff
@@ -18,22 +18,22 @@ fr:
     latest: Récent
     like: Aime
     listings: Annonces
-    log_in: Entrer
+    log_in: Se connecter
     month: Mois
     more_from: Plus de
```

Comment:
You might want to change this one as well.
It is used in this context: 
![image](https://user-images.githubusercontent.com/50857179/135699777-5ce9aead-aeb2-44bf-b6d8-dfee81b61fd5.png)
Example in [page](https://dev.to/devteam/hacktoberfest-2021-is-here-4a3l).

Using "Plus de The DEV Team" would mean "more The DEV Team".
In this context, I would use "Plus de la part de The DEV Team".